lighthouse-core/audits/bootup-time.js

```diff
@@ -124,6 +129,13 @@ class BootupTime extends Audit {
       .filter(result => result.total >= context.options.thresholdInMs)
       .sort((a, b) => b.total - a.total);
 
+
+    // TODO: consider moving this to core gathering so you don't need to run the audit for warning
+    if (hadExcessiveChromeExtension) {
+      runWarnings.push('Chrome extensions impacted the performance of this page. ' +
+        'Try running Lighthouse in a clean Chrome profile to avoid being impacted by extensions.');
```

Comment:
@kaycebasques wdyt about this wording? "impacted" feels a little awkward to me, but I could roll with it. 

I think we should add ", or incognito mode, " to the second sentence though.

Reply:
"Impacted" doesn't jump out to me, but since you summoned me:

* Interfered
* Negatively affected
* Distorted

Chrome extensions negatively affected this page's load performance. Try auditing the page in incognito mode or from a clean Chrome profile.

(I think you can drop "to avoid being impacted by extensions"... it's clear enough why you'd use a clean profile or incognito)

Reply:
good feedback @kaycebasques I like it! done